lua/telescope/_extensions/project_actions.lua

```diff
@@ -0,0 +1,61 @@
+local builtin = require("telescope.builtin")
+local actions = require("telescope.actions")
+local transform_mod = require('telescope.actions.mt').transform_mod;
+
+local project_actions = {}
+
+local project_dirs_file = vim.fn.stdpath('data') .. '/telescope-projects.txt'
+
+project_actions.add_project = function()
+	local git_root = vim.fn.systemlist("git -C " .. vim.loop.cwd() .. " rev-parse --show-toplevel")[1]
+	if not git_root then
+		print('Telescope-project adds projects based on a git root. No project found. Please refer to documentation to add projects manually')
+		return
+	end
```

Comment:
You can try to get git root and if not possible just use current directory

Reply:
Yeah that's a good idea, thanks